src/shared/stores/widgetLayerSlice.ts

```diff
@@ -0,0 +1,55 @@
+import { StateCreator } from 'zustand';
+import { BoundSlice } from './backgroundWidgetLayerStore';
+
+type Layout = {
+  i: string;
+  x: number;
+  y: number;
+  w: number;
+  h: number;
+};
+
+type Position = {
+  breakpoints: string;
+  layouts: { [key: string]: Layout[] };
+};
+
+const WIDGET_KEYS = {
```

Comment:
```suggestion
export const WIDGET_KEYS = {
```

이부분을 export로 바꾸고 WidgetLayer에서 위젯 별 키 값을 이걸로 가져다 쓰면 될 것 같습니당

Reply:
[548c7d5](https://github.com/PomPom-Home/PomPomHome/pull/19/commits/548c7d54bd26a19049bba88ab79569f3142b85ec) ✅